contributing/building-a-bento-amp-extension.md

```diff
@@ -64,11 +64,12 @@ The directory structure is below:
 |   ├── test/                            # Element's unit test suite (req'd)
 |   |   ├── test-amp-my-element.js
 |   |   └── More test JS files (optional)
-|   ├── my-element.js                    # Preact implementation (req'd)
-|   ├── my-element.type.js               # Preact type definitions (req'd)
+|   ├── base-element.js                  # Preact base element (req'd)
+|   ├── component.js                     # Preact implementation (req'd)
+|   ├── component.type.js                # Preact type definitions (req'd)
 |   ├── amp-my-element.js                # Element's implementation (req'd)
 |   ├── amp-my-element.css               # Custom CSS (optional)
```

Comment:
A component now could technically have AMP CSS and regular CSS now, do we have an example of how this would be structured? I assume the AMP CSS could import the regular CSS and register it with `AMP.registerExtension`, and the regular CSS file would be the import provided with the script for Bento documents?

Reply:
The terminology around the different bits of CSS is a little ambiguous to me here, but my sense is that a component should use the `.jss.js` and import w/ `useStyles`, while the `amp-my-element.js` should import the `build/*/*.css` file and provide that CSS via `registerElement`.